src/ui/footer/Navigation.tsx

```
@@ -0,0 +1,141 @@
+
+  return (
+    <nav 
+      className="flex flex-wrap items-start gap-x-12 gap-y-6 max-sm:flex-col"
+      aria-label="Footer navigation"
+    >
+      {footerMenu?.items?.map((item, key) => {
+        switch (item._type) {
+          case 'link':
+            if (item.external) {
+              return (
+                <div className="flex flex-col gap-2" key={key} role="navigation">
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<nav>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)